miner/worker.go

```diff
@@ -187,30 +188,34 @@ type worker struct {
 
 	// Needed for randomness
 	db *ethdb.Database
+
+	// Gauge counting the block finalization time(from created to finalized)
+	blockFinalizationTime metrics.Gauge
```

Comment:
Instead of making this a field of `worker`, can it be made as a package level variable, like this:  https://github.com/celo-org/celo-blockchain/blob/master/core/blockchain.go#L52

Reply:
Agree, that makes more sense